packages/sdk-client-v3/src/utils/executor.ts

```diff
@@ -33,13 +33,15 @@ export default async function executor(request: HttpClientConfig) {
 
   const data: TResponse = await executeHttpClientRequest(
     async (options: HttpClientConfig): Promise<TResponse> => {
-      const { enableRetry, retryConfig } = rest
+      const { enableRetry, retryConfig, abortController } = rest
       const {
         retryCodes = [],
         maxDelay = Infinity,
         maxRetries = 3,
         backoff = true,
         retryDelay = 200,
+        // If set to true reinitialize the abort controller when the timeout is reached and apply the retry config
+        retryOnAbort = false,
```

Comment:
Was there any reason why this was made `false` by default?

Reply:
It's for consistency with v2: https://github.com/commercetools/commercetools-sdk-typescript/blob/68ad0e322b355cdee28e3114c95e60d0b9191df0/packages/sdk-client/src/sdk-middleware-http/http.ts#L91

Reply:
Okay,  but since this is a new Major Version, I think some things have to change. I believe making this option true by default is better. Abort operation is usually network related error, so regardless of whether v2 is false by default I think it will make more sense to have this true here.

What do you think?

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/714/commits/3c40d1de696e972de14c27a4dfc2e00b7af00c7c, https://github.com/commercetools/commercetools-docs/pull/4986/commits/a981b6b9b3bb15d679593f377fad85abea1fb86a